lighthouse-core/test/gather/gatherers/seo/font-size-test.js

```diff
@@ -93,21 +127,21 @@ describe('Font size gatherer', () => {
     };
 
     const artifact = await fontSizeGather.afterPass({driver});
-    const expectedFailingTextLength = Array.from(smallText.trim()).length;
-    const expectedVisitedTextLength = Array.from(bigText.trim()).length + expectedFailingTextLength;
```

Comment:
this didn't make any sense so I changed it.

Reply:
well `failingText` as a test doesn't really make any sense at all anymore because we don't visit nodes individually anymore, so either the whole thing fails or it passes, there's no individual node failure recovery. we should just kill it.

aside: I think you've rewritten `failingText` in these tests to be "text that is too small" but that was supposed to be `smallText` which might be why these lengths were so confusing to you (that and the fact that they are very confusingly named 😄)